Backend/dev/migrations/dynamic-offer-driver-app/0484-added-pn-and-payout-vpa-status.sql

```diff
@@ -0,0 +1,34 @@
+ALTER TABLE atlas_driver_offer_bpp.driver_information ADD COLUMN payout_vpa_status Text;
```

Comment:
It is repeated, column is already there.

Reply:
removed